src/components/app/App.js

```diff
@@ -51,17 +52,19 @@ class App extends Component {
         case option2:
           this.onSortByAuthorClick(); // вызываем сортировку по автору, если 2 элемент массива sortingOptions
           this.setState({
-            selectedOption: option1 // записываем в стейт выбранную опцию
+            selectedOption: option2 // записываем в стейт выбранную опцию
```

Comment:
As you are setting `selectedOption` inside `onSortByDefault` method you can even remove `setState` from lines 54-56
it would be even better approach